en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Offboard mode is primarily used for controlling vehicle movement and attitude, and supports only a very limited set of MAVLink commands (more may be supported in future). It can be used to:
+* Control vehicle position or velocity (`SET_POSITION_TARGET_LOCAL_NED`). 
+* Control vehicle attitude/orientation (`SET_ATTITUDE_TARGET`)
+
+> **Note** Velocity and attitude commands must be resent regularly at a rate of around (TBD Hz) or PX4 will stop the vehicle. 
+
+Other operations, like taking off, land, return to launch, arming, are best handled using the appropriate modes. Operations like uploading, downloading missions can be performed in any mode.

```

Comment:
Takeoff and land is also possible with the appropriate bitmasks.

Reply:
@julianoes recommended just using the supported modes for that. I tend to agree - lots of thought has gone into landing/takeoff on various vehicles, so while you can manually reinvent the wheel by sending appropriate setpoints, easier to just invoke "takeoff" and "land" mode.
Do both of you think perhaps this would be better as:
> other operations, like taking off, land, return to launch, arming, are best handled using the appropriate modes (you can takeoff and land using setpoint commands, but the respective modes are safer and easier to use).

Reply:
Yes agreed